python/tests/utils/test_bin_packing.py

```diff
@@ -41,7 +41,7 @@ def weight_func(x: int) -> int:
         return max(x, open_cost)
 
     item_list_sums: List[int] = [sum(item) for item in PackingIterator(splits, split_size, lookback, weight_func)]
-    assert all([split_size >= item_sum >= 0 for item_sum in item_list_sums])
+    assert all(split_size >= item_sum >= 0 for item_sum in item_list_sums)
```

Comment:
This is a very nice check because it will exit the loop when one `false` has been encountered.